File: program/convert-string-to-lowercase/ConvertStringToLowercase.java
import java.util.Scanner;

public class ConvertStringToLowercase {
	
	public static String ConvertStringToLowercase(String s) {
		return s.toLowerCase();
	}

	public static void main(String[] args) {
		 Scanner scanner = new Scanner(System.in);
		 String s = scanner.nextLine();
		 System.out.println(ConvertStringToLowercase(s));
	}

}

File: program/program/convert-string-to-lowercase/ConvertStringToLowercase.kt
import java.util.Scanner

fun main() {
    print("Enter String : ")
    println()
    var input = readLine()

    println("Lower Case: " + ConvertStringToLowercase(input!!))

}

private fun ConvertStringToLowercase(args: String): String {
    return args.lowercase().toString()
}